.coveralls.yml

```diff
@@ -0,0 +1 @@
+repo_token: 6ggWHxHTG2JvepQ98oe4baMzh7EWPOtDM
```

Comment:
@tomuhenry Could you add this as an environment variable. Thanks.

Reply:
Ok... Working on it